cli_client/python/timesketch_cli_client/commands/intelligence.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2023 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Commands for interacting with intelligence within a Sketch."""
+
+import sys
+import json
+import click
+
+
+@click.group("intelligence")
+def intelligence_group():
+    """Manage intelligence within a sketch."""
+
+
+@intelligence_group.command("list")
+@click.option(
+    "--header/--no-header",
+    default=True,
+    help="Include header in output. (default is to show header))",
+)
+@click.option(
+    "--columns",
+    default="ioc,type",
+    help="Comma separated list of columns to show. (default: ioc,type)",
+)
+@click.pass_context
+def list_intelligence(ctx, header, columns):
+    """List all intelligence.
+
+    Args:
+        ctx: Click context object.
+        header: Include header in output. (default is to show header)
+        columns: Comma separated list of columns to show. (default: ioc,type)
+                Other options: externalURI, tags
+    """
+
+    if not columns:
+        columns = "ioc,type"
+
+    columns = columns.split(",")
+
+    output = ctx.obj.output_format
+    sketch = ctx.obj.sketch
+    try:
```

Comment:
This `try` is pretty broad. Can you move this into a guard instead?

Reply:
actually I just moved it around the get_intel, that is the only one it matters here